App.js

```diff
@@ -6,38 +6,488 @@ import styles from './src/style';
 import RouteSelection from './src/components/RouteSelection';
 
 const url = 'https://sase-labs-2020.github.io/assets/directions/coffman_yudof.json'
+import MapView, {Marker} from 'react-native-maps';
 
-export default class App extends Component {
-	constructor(props) {
-		super(props);
-		this.state = {
-			loaded: false,
-		}
-	}
-
-	async componentDidMount() {
-		await Font.loadAsync({
-			'neutraface': require('./assets/fonts/NeutraText-Bold.otf'),
-			'open-sans': require('./assets/fonts/OpenSans-Regular.ttf')
-		});
-		this.setState({loaded: true});
-	}
-
-	render() {
-		if (!this.state.loaded) {
-			return (
-				<View>
-					<ActivityIndicator/>
-				</View>
-			);
-		}
-
-
-		return (
-			<View style={styles.container}>
-				<RouteSelection/>
-			</View>
-		);
-	}
+export default class BuildingMarker extends Component {
+  
+  constructor(props){
+          super(props);
+          this.state = {isLoading: true, scrollable: false , data:[]};
+     }
+  
+     componentDidMount(){
+         var paths = this.props.filenames.reduce(
+  ["10Church",
```

Comment:
Mentioned this at the meeting. Please see [this line](https://github.com/SASE-Labs-2020/gopherway/blob/master/src/components/GraphEdge.js#L36) to get an idea of how to store the names and avoid hard-coding details.